etc/web-streams-polyfill.api.md

```diff
@@ -128,12 +128,13 @@ export type ReadableStreamDefaultController<R> = ReadableStreamDefaultController
 export type ReadableStreamDefaultReader<R> = ReadableStreamDefaultReader_2<R>;
 
 // @public (undocumented)
-export interface ReadResult<T = any> {
-    // (undocumented)
-    done: boolean;
-    // (undocumented)
+export type ReadResult<T> = {
```

Comment:
This aligns with TypeScript 3.8's [ReadableStreamReadDoneResult and ReadableStreamReadValueResult](https://github.com/microsoft/TypeScript/blob/v3.8.2/lib/lib.dom.d.ts#L1533-L1541).